packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -94,7 +94,10 @@ class CodeSnippetDisplay extends React.Component<ICodeSnippetDisplayProps> {
       const documentWidget = widget as DocumentWidget;
       const fileEditor = (documentWidget.content as FileEditor).editor;
       const markdownRegex = /^\.(md|mkdn?|mdown|markdown)$/;
-      if (PathExt.extname(widget.context.path).match(markdownRegex) !== null) {
+      if (
+        PathExt.extname(widget.context.path).match(markdownRegex) !== null &&
+        snippet.language.toLowerCase() !== 'markdown'
```

Comment:
you also need to add a check on line 124 below for markdown cells in notebooks

Reply:
Should we add `markdown` and `python` into `JupyterLab Contents`?

Reply:
@lresende what do you mean by `JupyterLab Contents`?